### DISCUSSION TITLE
Symphonia release cycle

### DISCUSSION THREAD

**Discussion by mxw-meta** [pdeljanov/Symphonia#256] [General]
Dear Symphonia Developers,
Dear @pdeljanov,

I hope this message finds you well. 
As a user of Symphonia I have been following its development with interest. To align my usage, and potential contributions with your project's release schedule, I was hoping to gather some information regarding your release cycle.

- How frequently do you typically release new versions or updates? i.e. do you aim for X releases each year?
- Is there a planned schedule for an upcoming releases?

Thank you for your time and dedication to this project. I appreciate your efforts and look forward to your response!

Best regards,
Max

**Comment by pdeljanov** [comment]
Hi @mxw-meta,

There's no official schedule. It very much is a "when I have time & updates" sort of thing. However, if something is very badly broken, I'll cut a release quickly to address that (though, nothing major has come up).

Right now I'm in the process of catching up on PRs and Issues. I will likely release a 0.5.4 in the next month or so. Hoping to do a 0.6 sometime later as well to address some API pain points.

---

I will plug #203 though for anyone reading. I am looking for an additional contributors who would like to help triage/fix issues, and test/review PRs.

**Reply by mxw-meta** [reply]
Thanks for the reply and for driving this project @pdeljanov 
Great to hear a new release is on the horizon. For what it's worth, I'm mainly interested in getting AIFF support and the malformed header fix in a stable version.

Thanks for pointing out the call for co-maintainers. I'll evaluate my time and see how useful I could be to support.

**Comment by Jaxx497** [comment]
Hey, hate to bump a super old thread, but I've noticed quite a bit of work has been done on the dev 0.6 branch as well as some recent PRs. Wondering if there's any projections on a 0.6 release? (And again, thank you for all the work you've done)

**Reply by pdeljanov** [reply]
Hey @Jaxx497,

I figured it'd be more useful to everyone if I opened an issue to track this. Please see #330 .
